index.coffee

```diff
@@ -48,16 +58,52 @@ exports.del = (opts, cb) ->
 	opts.method = 'DELETE'
 	exports(opts, cb)
 
+# Convert regex to string, remove leading/trailing '/' and use as object key.
+# The result object can easily be converted back to a regexp with `new RegExp()`
+regexToStr = (regex) ->
```

Comment:
I'm not sure, but I think this could be replaced with `regex.source` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/source)?

Reply:
@afitzek I totally reinvented the wheel here, fixed